docs/src/main/asciidoc/building-my-first-extension.adoc

```diff
@@ -861,7 +861,7 @@ Run the application and notice the `Installed Features` list contains the `greet
 
 [source,shell,subs=attributes+]
 ----
-$ mvn clean compile quarkus:dev
```

Comment:
Kind of minor but the logs below are for `mvn clean compile quarkus:dev`.

Reply:
updated it - interesting to see how more "noisy" our output become in a year ;)